tests/admin_checks/tests.py

```diff
@@ -138,6 +143,86 @@ class SongAdmin(admin.ModelAdmin):
         ]
         self.assertEqual(errors, expected)
 
+    def test_wrong_prepopulated_fields(self):
+        class SongAdmin(admin.ModelAdmin):
+            prepopulated_fields = {'title': 'test'}
+
+        expected = [
+            checks.Error(
+                'The value of \'prepopulated_fields["title"]\' must be a list'
+                ' or tuple.',
+                obj=SongAdmin,
+                id='admin.E029',
+            )
+        ]
+        self.assertEqual(SongAdmin(Song, AdminSite()).check(), expected)
+
+    def test_wrong_readonly_fields(self):
+        class SongAdmin(admin.ModelAdmin):
+            readonly_fields = 'test'
+
+        expected = [
+            checks.Error(
+                "The value of 'readonly_fields' must be a list or tuple.",
```

Comment:
Double quotes are used in this case because single quotes are part of the text. In order to switch single quotes to double quotes in the text if that's what you want we would have to change the original text of the error which is not the goal of this PR.